### ISSUE TITLE
USER ISSUE: Disappearing Dirt Blocks

### ISSUE THREAD

**Issue by Sebi92** [StrangeLoopGames/EcoIssues#5927]
**Version:** 0.7.2.3 beta

**Steps to Reproduce:**
I terraformed my land a bit and filled little holes with dirt
But they disappeared after a while (I wasn't near the location)
That happened not only once

**Expected behavior:**

**Actual behavior:**

**Comment by MarjieAVolk** [comment]
Closing as duplicate of https://github.com/StrangeLoopGames/EcoIssues/issues/5900
